README.md

```diff
@@ -1,14 +1,10 @@
 # netatmo-weather-adapter
 Netatmo Weather Station adapter for the [Mozilla IoT gateway](https://iot.mozilla.org).
 
-Bridges the Netatmo REST API to the gateway so it can read data from Netatmo Weather Stations (and Health Coaches).
+Bridges the Netatmo REST API (via netatmo.com, not the local Netatmo device) to the gateway so it can read data from Netatmo Weather Stations (and Health Coaches).
```

Comment:
I don't think that's the proper clarification. If anything I'd call it the Netatmo Cloud REST API or similar.